ArjunaCore/arjuna/classes/com/arjuna/ats/arjuna/tools/osb/mbean/UidWrapper.java

```diff
@@ -209,4 +209,8 @@ public boolean isAllowRegistration() {
 	public void setAllowRegistration(boolean allowRegistration) {
 		this.allowRegistration = allowRegistration;
 	}
+
+	public void setMBean(OSEntryBean mbean) {
```

Comment:
@marcosgopen _maybe_ we consider to move this to package local access? What do you think? UidWrapper is part of the public API of Narayana and I am not sure we necessarily want to encourage users to try to set this themselves?

Reply:
Sure, thanks